src/modules/GroupChannel/components/SuggestedReplies/index.scss

```diff
@@ -47,14 +47,14 @@
 }
 
 .sendbird-suggested-replies__option {
-  white-space: nowrap;
-  height: 32px;
+  white-space: pre-wrap;
   font-size: 12px;
-  padding: 0 14px;
+  padding: 8px 14px;
```

Comment:
There could be extra padding(or margin) in Widget which used to be covering this. If so, please remove it :)

Reply:
no widget now uses UIKit SR so no problem